test/unit/checkbox_question_presenter_test.rb

```diff
@@ -13,6 +13,7 @@ class CheckboxQuestionPresenterTest < ActiveSupport::TestCase
       @renderer.stubs(:option).with(:option1).returns("Option 1")
       @renderer.stubs(:option).with(:option2).returns({ label: "Option 2", hint_text: "Hint 2" })
       @renderer.stubs(:option).with(:option3).returns({ label: "Option 3" })
+      @renderer.stubs(:show_caption?).returns(true)
```

Comment:
Is this needed if the default behaviour is to return `true`?

Reply:
So unfortunately, you need to stub the method because "@renderer" in the context is just a stub object, not actually a renderer. We need to stub the hide_caption method because it's called when the we test `presenter.caption`. But I'll move the stub closer to where its needed (L39)